specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/ManagedEnvironments_CreateOrUpdate.json

```diff
@@ -35,7 +60,27 @@
               "customerId": "string"
             }
           },
-          "zoneRedundant": true
+          "zoneRedundant": true,
+          "workloadProfiles": [
+            {
+              "workloadProfileType": "GeneralPurpose",
+              "provisioningState": "Succeeded",
```

Comment:
this is for [ManagedEnvironments_CreateOrUpdate.json](https://github.com/Azure/azure-rest-api-specs/pull/19770/files#diff-6f12fcd602a58a7be78a0f0b9719c4b9aa35cb76966a2ca2dc12201129d941ba), so should no provisioningState